docs/UserGuide.md

```diff
@@ -145,31 +97,28 @@ Examples:
 * `list` followed by `delete 2` removes 2nd show in Trackermon
 * `find ghibli` followed by `delete 1` removes 1st show in results of `find` command
 
-### Clearing all entries : `clear`
-
-Clears all entries from the address book.
-
-Format: `clear`
 
 ### Exiting the program : `exit`
 
-Exits the program.
+Exits the program. 
 
 Format: `exit`
 
-### Saving the data
+* Displays error message and exits the programme after 3 seconds.
```

Comment:
Should it be program instead of programme?

Reply:
Fixed